### PR TITLE
[MRG] DOC: update docstring

### PR DESCRIPTION
**Pull request by jasmainak**
Fixes https://github.com/mne-tools/mne-bids/pull/135/files#r261892123

### CODE REVIEW

File: mne_bids/io.py [hunk 1]
```diff
@@ -105,9 +105,22 @@ def read_raw_bids(bids_fname, bids_root, return_events=True,
     bids_root : str
         Path to root of the BIDS folder
     return_events: bool
```
Comment (monkeyman192) [inline mne_bids/io.py]: should add `, optional` I think

File: mne_bids/io.py [hunk 2]
```diff
@@ -105,9 +105,22 @@ def read_raw_bids(bids_fname, bids_root, return_events=True,
...
+        Whether to return events (and event_id) or not.
+        Default is True.
     verbose : bool
         The verbosity level
+
+    Returns
+    -------
+    raw : instance of Raw
+        The MNE raw object containing the data.
+    events : array, (n_events, 3)
```
Comment (monkeyman192) [inline mne_bids/io.py]: Should there be an `, optional` for the two parameters that are only returned if `return_events` is `True`?
Also this raises a question... Is it good to have a function that can return differing numbers of objects?

File: mne_bids/io.py (hunk 1 above)
Comment (jasmainak) [reply mne_bids/io.py]: well all keyword arguments are optional. The function signature will tell you if it's optional or not.

Comment (monkeyman192) [reply mne_bids/io.py]: fair enough, I just thought the spec for the numpy docstring said to have the `optional` tag in it if they were kwargs

File: mne_bids/io.py (hunk 2 above)
Comment (jasmainak) [reply mne_bids/io.py]: I've seen [this kind of code in MNE](https://github.com/mne-tools/mne-python/blob/73ea718b82c7ea5f09b70490ca3ba7de2fd79f78/mne/time_frequency/multitaper.py#L139). I'm not sure if it's a good idea in public API but I will defer to the opinion of @agramfort :)

File: mne_bids/io.py (hunk 1 above)
Comment (jasmainak) [reply mne_bids/io.py]: ah indeed, I learned something today :)

File: mne_bids/io.py (hunk 2 above)
Comment (sappelhoff) [reply mne_bids/io.py]: I haven't seen any indication that an `, optional` should be used for multiple return values (some of which are optional).

That apart, I do think it's a confusing practice, see also [this discussion](https://grokbase.com/t/python/python-list/028wpwxmvt/stylistic-question-optional-return-value/oldest)

I think it'd be more helpful if the same amount of outputs is always returned ...

File: mne_bids/io.py (hunk 1 above)
Comment (sappelhoff) [reply mne_bids/io.py]: I can confirm, see [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) and search for `, optional`

File: doc/index.rst [hunk 3]
```diff
@@ -14,7 +14,7 @@ Installation
 We recommend the `Anaconda Python distribution <https://www.continuum.io/downloads>`_ and Python 3.
 
 Besides numpy, scipy and matplotlib (which are included in the standard Anaconda installation), 
-you will need to install the most recent version of MNE using the `pip` command line tool:
+you will need to install the most recent version of MNE using the `pip` command line tool::
```
Comment (sappelhoff) [inline doc/index.rst]: Why the double colon `:` ? 
```suggestion
you will need to install the most recent version of MNE using the `pip` command line tool:
```

**Comment by sappelhoff** [comment]
This was taken care of in #191 #195

**Pull request closed without merging**